### ISSUE TITLE
Error parsing timestamps in format mm:ss 

### ISSUE THREAD

**Issue by santiq** [Claviz/xlstream#30]
Hello

I have the following value in an xls file: 

`14:58.3` 

but when is parsed, both `raw` and `formatted` results are wrong

![Screen Shot 2020-10-19 at 10 08 07](https://user-images.githubusercontent.com/7070683/96456052-5f177f80-11f4-11eb-8ef1-02fa1d954e67.png)

Is there a way that I could just get a string with "14:54.3" raw value?

Here is the xlsx file:
[timestamp_error.xlsx](https://github.com/Claviz/xlstream/files/5402059/timestamp_error.xlsx)

**Comment by jansivans** [comment]
Hi.
Internally this cell has a number format with the code `47`.
According to [this ](https://www.ecma-international.org/news/TC45_current_work/Office%20Open%20XML%20Part%204%20-%20Markup%20Language%20Reference%20(tagged).pdf) document (page 2128), code `47` equals to `mmss.0` format, so it is parsed correctly according to the specification.
This specification also has a note about number formatting:
_Some of these Ids are interpreted differently, depending on the UI language of the implementing application._
So my guess is that Excel (or other app you are using to view XLSX files) interprets this number differently and that's why you see `14:58.3` instead of `1458.3`.

**What can you do**
You have two ways of solving your issue:

1. Implement a function which gets parsed value and adds a colon symbol in correct place.
2. Get raw number value and format it to `mm:ss.0` using some formatting library like [numfmt](https://github.com/borgar/numfmt)

Have I answered your question?

**Comment by santiq** [comment]
Hey @jansivans thanks for this quick and detailed answer, you know a lot about the XML standard.

> Internally this cell has a number format with the code 47.

How did you get the format number of the cell? 

If I could get that, I could go for option 1)
> Implement a function which gets parsed value and adds a colon symbol in correct place.


But I need that information since I'm using this library to parse a lot of different xls and adding always the colon symbol would be tricker without this information could break my other use cases.

**Comment by jansivans** [comment]
I have found that my theory is right.
[Excel uses non-standard formatting](https://docs.microsoft.com/en-us/openspecs/office_standards/ms-oi29500/17d11129-219b-4e2c-88db-45844d21e528):
![image](https://user-images.githubusercontent.com/5667073/96543406-fc35ef00-12ac-11eb-8cc5-963c64e3bf3b.png)

Anyway, I have solved your problem by adding a new option `numberFormat`.
You can pass `excel` string to it and everything will be formatted according to `Excel` specification:
```js
const { getXlsxStream } = require('xlstream');

(async () => {
    const stream = await getXlsxStream({
        filePath: './source.xlsx',
        sheet: 0,
        numberFormat: 'excel'
    });
    stream.on('data', x => console.log(x));
})();
```
This option is available starting from `2.3.0` version of `xlstream`.
Please test and let me know if it works for you.

**Comment by santiq** [comment]
@jansivans Thanks it worked!! 

You saved my day mate, I owe you a beer 🍺
